lms/djangoapps/course_api/views.py

```diff
@@ -20,12 +26,13 @@ class CourseView(APIView):
     def get(self, request, course_key_string):
         """
         Request information on a course specified by `course_key_string`.
-            Body consists of a `blocks_url` that can be used to fetch the
-            blocks for the requested course.
+
```

Comment:
Need to update the docstring to include the additional fields that are now returned in the `course_detail` view.